src/expr.h

```diff
@@ -50,6 +50,7 @@ enum
   NEG,
   IFNEG,
   IFZERO,
+  RAT_CAST,
```

Comment:
I think I would prefer this to be called something that closely mimicked the name of the operator, perhaps `MPZ_TO_MPQ` ?

Reply:
Seems reasonable. I'll do this.

Reply:
Done in commit [`d3b27b4`](https://github.com/CVC4/LFSC/pull/29/commits/d3b27b41147ecee6e75c99bd41426746e1a80f13).